data/skeletons/lalr1.cc

```diff
@@ -1194,7 +1218,100 @@ b4_dollar_popdef])[]dnl
   {
     error (]b4_join(b4_locations_if([yyexc.location]),
                     [[yyexc.what ()]])[);
-  }]b4_lac_if([[
+  }]b4_parse_error_bmatch([verbose\|detailed\|custom], [[
+
+  int
+  ]b4_parser_class[::Context::yyexpected_tokens (int yyarg[], int yyargn) const
+  {
+    // Actual number of expected tokens
+    int yycount = 0;
+]b4_lac_if([[
+#if ]b4_api_PREFIX[DEBUG
+    // Execute LAC once. We don't care if it is succesful, we
```

Comment:
"Successful".  But the error was already there, and several times.  Could you please address them too?
```
data/skeletons/lalr1.cc:        // Execute LAC once. We don't care if it is succesful, we
etc/bench.pl.in:Display this message and exit succesfully.  The more verbose, the more
src/location.c:   Return whether succesful.*/
tests/calc.at:# third and fourth are.  Parse status is succesful.
```

Reply:
addressed in an additional separate commit